src/random_planet.py

```diff
@@ -1,5 +1,5 @@
 import random
 
-planets = ['Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune', 'Pluto', 'Moon']
+planets = ['Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune', 'Pluto', 'Moon', 'Titan']
```

Comment:
Titan? Is that a planet? And what's up with the moon and pluto? Maybe think about making a change to remove all of those.